acceptance-tests/dsl/src/main/java/org/hyperledger/besu/tests/acceptance/dsl/node/ProcessBesuNodeRunner.java

```diff
@@ -421,6 +421,29 @@ public void startNode(final BesuNode node) {
                 + "acceptance-tests/tests/build/resources/test/acceptanceTesting.security");
     // add additional environment variables
     processBuilder.environment().putAll(node.getEnvironment());
+
+    try {
+      Integer debugPort = Integer.parseInt(System.getenv("BESU_DEBUG_CHILD_PROCESS_PORT"));
+      if (debugPort != null) {
+        LOG.warn("Waiting for debugger to attach to SUSPENDED child process");
+        String debugOpts =
+            " -agentlib:jdwp=transport=dt_socket,server=y,suspend=y,address=*:" + debugPort;
+        String prevJavaOpts = processBuilder.environment().get("JAVA_OPTS");
+        if (prevJavaOpts == null) {
+          processBuilder.environment().put("JAVA_OPTS", debugOpts);
+        } else {
+          processBuilder.environment().put("JAVA_OPTS", prevJavaOpts + debugOpts);
+        }
+
+      } else {
+        LOG.debug(
+            "Child process may be attached to by specifying -DBESU_DEBUG_CHILD_PROCESS_PORT=<port>");
```

Comment:
-D specifies a System property, not an envar.  if you intend for it to be a system property we should use System.getProperty rather than System.getenv (above)